source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,23 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+You can watch for changes in MongoDB by using the `watch()
+<https://mongodb.github.io/node-mongodb-native/3.5/api/Collection.html#watch>`_ method. ``watch()`` opens a :manual:`Change Stream</changeStreams/>`, that allows applications to watch for changes to data and react to those changes. After opening a ``Change Stream``, you can process the change events using the `changeStream.next()
+<http://mongodb.github.io/node-mongodb-native/3.5/api/ChangeStream.html#next>`_ method. ``watch()`` takes an optional ``pipeline`` parameter that is an array of :manual:`stages </reference/operator/aggregation-pipeline/#stages>` to filter the change events output. You can close a change stream 
```

Comment:
I wonder if it might be more clear to define what it means to close a change stream here. For example:
"You can stop watching for changes and free the related resources by closing the change stream using the ..."

Reply:
I've altered your suggestion slightly to: 

"You can stop watching for changes and free resources by closing the change stream using the..."

I think that makes it very clear to a Jasmine-level reader